pkg/volume/config_data/doc.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package config_data contains the implementation of ConfigDataVolumeSource
+package config_data
```

Comment:
All package names should be lower case, no separators (only a-z0-9)

On Tue, Oct 13, 2015 at 1:43 AM, Tamer TAS <notifications@github.com> wrote:

> In pkg/volume/config_data/doc.go
> <https://github.com/kubernetes/kubernetes/pull/6245#discussion_r41828856>:
>
> > +
> > +Licensed under the Apache License, Version 2.0 (the "License");
> > +you may not use this file except in compliance with the License.
> > +You may obtain a copy of the License at
> > +
> > +    http://www.apache.org/licenses/LICENSE-2.0
> > +
> > +Unless required by applicable law or agreed to in writing, software
> > +distributed under the License is distributed on an "AS IS" BASIS,
> > +WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +See the License for the specific language governing permissions and
> > +limitations under the License.
> > +*/
> > +
> > +// Package config_data contains the implementation of ConfigDataVolumeSource
> > +package config_data
>
> @ncdc <https://github.com/ncdc> oh, my mistake you are correct. The
> underscored package name caused the problem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/6245/files#r41828856>.
>